src/run/runner/wall_time/helpers.rs

```diff
@@ -0,0 +1,272 @@
+use crate::prelude::*;
+use runner_shared::walltime_results::WalltimeResults;
+use std::path::Path;
+
+fn add_error_explanation(error_details: &str) -> String {
+    format!(
+        "The command did not produce any CodSpeed result to process, did you run your benchmarks using a compatible CodSpeed integration?\n\
+        Check out https://codspeed.io/docs/benchmarks/overview for more information.\n\n\
+        Details: {error_details}"
+    )
+}
```

Comment:
- [ ] Update the name of the function to something more explanatory, like "add_empty_results_error_explanation"